src/software/ai/hl/stp/tactic/receiver/receiver_fsm.cpp

```diff
@@ -74,29 +74,25 @@ std::optional<Shot> ReceiverFSM::findFeasibleShot(const World& world,
     // The percentage of open net the robot would shoot on
     if (best_shot_opt)
     {
-        // Vector from the ball to the robot
         Vector robot_to_ball = world.ball().position() - assigned_robot.position();
```

Comment:
accidentally deleted a comment here

Reply:
It seemed fairly intuitive with the variable name, so I thought it was a bit repetitive.